examples/promise-co-await/co.js

```diff
@@ -1,8 +1,8 @@
-var mysql = require('../promise.js');
+var mysql = require('../../promise.js');
```

Comment:
then it should be changed [everywhere](https://github.com/sidorares/node-mysql2/blob/master/examples/mysqlproxy.js)?